src/measure_theory/measure/finite_measure_weak_convergence.lean

```diff
@@ -154,6 +158,246 @@ def coe_add_monoid_hom : finite_measure α →+ measure α :=
+is `finite_measure.test_against'`. -/
+abbreviation test_against_nn (μ : finite_measure α) (f : α →ᵇ nnreal) : ℝ≥0 :=
+(lintegral (μ : measure α) ((coe : ℝ≥0 → ℝ≥0∞) ∘ f)).to_nnreal
+
+-- I believe the formulation is generally useful, except maybe the exact form
+-- of the assumption `f_bdd`.
+-- Where to place?
+lemma _root_.is_finite_measure.lintegral_lt_top_of_bounded_to_ennreal {α : Type*}
+  [measurable_space α] (μ : measure α) [μ_fin : is_finite_measure μ]
+  {f : α → ℝ≥0∞} (f_bdd : ∃ (c : ℝ≥0), ∀ (x : α), f(x) ≤ c) :
+  lintegral μ f < ⊤ :=
+begin
+  cases f_bdd with c hc,
+  have le : f ≤ (λ (x : α), (c : ℝ≥0∞)) := hc,
+  apply lt_of_le_of_lt (@lintegral_mono _ _ μ _ _ le),
+  rw lintegral_const,
+  exact ennreal.mul_lt_top ennreal.coe_lt_top.ne μ_fin.measure_univ_lt_top.ne,
+end
+
+-- I believe these two could be useful... (1) (Only the second one is used below, though.)
+-- Where to place?
+lemma _root_.nnreal.val_eq_dist_zero : ∀ (z : ℝ≥0), (z : ℝ) = dist 0 z :=
+by { intros z, simp only [nnreal.dist_eq, nnreal.coe_zero, zero_sub, nnreal.abs_eq, abs_neg], }
+
+-- I believe these two could be useful... (2)
+-- Where to place?
+lemma _root_.nnreal.val_eq_dist_zero' : ∀ (z : ℝ≥0), (z : ℝ) = dist z 0 :=
+by { intros z, have key := nnreal.val_eq_dist_zero z, rwa dist_comm at key, }
+
+-- Is there a simpler way for the following? I think this might be useful (it is used below).
+-- Where to place?
+lemma _root_.nnreal.le_add_dist (a b : ℝ≥0) : a ≤ b + (dist a b).to_nnreal :=
+begin
+  suffices : (a : ℝ) ≤ (b : ℝ) + (dist a b),
+  { have coe_eq : (coe : ℝ≥0 → ℝ) (b + (dist a b).to_nnreal) = (b : ℝ) + dist a b,
+    { rw nnreal.coe_add,
+      simp only [real.coe_to_nnreal', max_eq_left_iff, add_right_inj],
+      exact dist_nonneg, },
+    rw ←coe_eq at this,
+    apply nnreal.coe_le_coe.mp this, },
+  have key : abs (a-b : ℝ) ≤ (dist a b) := by refl,
+  linarith [le_of_abs_le key],
+end
+
+-- These two are specific but useful at least in this file... (1)
+private lemma continuous_bounded_nn_to_ennreal_comp_add {β : Type*} [topological_space β]
+  {f₁ f₂ : β →ᵇ ℝ≥0} :
+  (coe : ℝ≥0 → ℝ≥0∞) ∘ (f₁ + f₂) = ( ((coe : ℝ≥0 → ℝ≥0∞) ∘ f₁) + ((coe : ℝ≥0 → ℝ≥0∞) ∘ f₂)) :=
+by { funext x, simp only [ennreal.coe_add, pi.add_apply, function.comp_app], }
+
+-- These two are specific but useful at least in this file... (2)
+private lemma continuous_bounded_nn_to_ennreal_comp_smul {β : Type*} [topological_space β]
+  {c : ℝ≥0} {f : β →ᵇ ℝ≥0} :
+  (coe : ℝ≥0 → ℝ≥0∞) ∘ (c • f) = c • ( ((coe : ℝ≥0 → ℝ≥0∞) ∘ f)) :=
+begin
+  funext x,
+  simpa only [algebra.id.smul_eq_mul, function.comp_app, pi.smul_apply, ennreal.coe_mul],
+end
+
+-- This was useful for rewriting. Should it be kept as such or generalized?
+-- Where to place?
+lemma bounded_continuous_function.nnreal.smul_eq {β : Type*} [topological_space β]
+  {c : ℝ≥0} {f : β →ᵇ ℝ≥0} : (c • (f : β → ℝ≥0)) = ((c • f) : β →ᵇ ℝ≥0) := by refl
```

Comment:
I think this is `bounded_continuous_function.coe_smul`?  Although that lemma is written in a slightly odd way, maybe your way is better.

Reply:
Thank you! I had missed the existing lemma.

It turns out that with some other changes, both of these became unnecessary anyway, so I just removed this.